### ISSUE TITLE
Speed trap and accident/hazard reporting

### ISSUE THREAD

**Issue by Jomino1** [ClutchEngineering/Sidecar#91]
Another feature that would be amazing is the ability to report speed traps and accidents/hazards like pretty much every competitor now. The main difference is I think sidecar should give you alerts regardless if actively using turn by turn navigation, just like in Waze (unlike Apple and Google where you have to actively be using navigation to receive alerts). If you at some point added this feature, and satellite view, no one could touch sidecar!

**Comment by gr00gles** [comment]
Very much agreed

**Comment by cachatj** [comment]
this would be 🔥 

but afaik, two sources for this real-time data. 

1) Proprietary & expensive (if for sale) - think Waze, Google, Uber

2) Crowd sourced - cheaper but results depend on density of users in real-time 

if could be hobbled together using other features, but each one pushes ya farther from realtime

**Comment by klept0** [comment]
Wzsabre does this via crowd sourced - https://wzsabre.rocks/

**Comment by sidbmw** [comment]
> Wzsabre does this via crowd sourced - https://wzsabre.rocks/

Out of curiosity does this pull info from Waze?

**Comment by klept0** [comment]
Yes - I also use it with my radar detector app on Android called JBV1 or HighwayRadar as well.
